03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>The Daily Planet</title>
-        <link rel="stylesheet" href="css/daily-planet-style.css"/>
-    </head>
-    <body>
-        <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/daily-planet-style.css" />
+</head>
+
+<body>
+    <header>
+        <h1>The Daily Planet</h1>
+
+        <nav><a href="http://espn.com">Sports</a>
+            <a href="http://Do512.com">Local</a>
+            <a href="http://dowjones.com">Economy</a></nav>
+    </header>
+
+    <main>
+        <section>
+            <article>
+                <h1>Giant Hello Kitty Attacks!!</h1>
+                <img src="https://s-media-cache-ak0.pinimg.com/564x/5e/9c/90/5e9c904179d02da70e80ea252219d0fe.jpg" />
+
+                <h5><em>written by: Lois Lane</em><h5>
+                  <p>In an amazing show of manliness, Superman has once again saved our city! Early yesterday morning, a large
+                    white giant kitten (aka "Hello Kitty") waddled down our Main Street.</p>
+                    <p>Superman, hearing the invasion with his super hearing, spead to the scene of <thead>
+                      litter box and lured the kitty away with a lure of catnip.</p>
+
+
+            </article>
+        </section>
+        <aside><img src="https://ll-us-i5.wal.co/asr/cf754d7a-a16e-45b0-8026-b1854d847a95_1.210d9d1e3fc51f14e8d1fe5d8aea9594.jpeg?odnHeight=450&odnWidth=450&odnBg=FFFFFF" />
+      </aside>
+        <footer>This is the footer. Insert a table here with columns for the various social media links</footer>

```

Comment:
Barbette,

Great job using all the elements required and picking a fun news story that ties into the theme of the assignment. The only thing missing from the assignment requirements was:

"Add copyright of company and current year in footer".

Other than that, good job!